public/prism.css

```diff
@@ -0,0 +1,95 @@
+code[class*="language-"],
+pre[class*="language-"] {
+  color: var(--syntax-text);
+  text-shadow: 0 1px rgba(0, 0, 0, 0.1);
+  font-family: Inconsolata, Monaco, Consolas, "Courier New", Courier, monospace;
```

Comment:
**issue:** `Inconsolata` is not a default font for most web-browsers and operating systems to my knowledge. 

**issue:** We are currently using [PT Mono](https://fonts.google.com/specimen/PT+Mono) for monospaced typography. I suggest we continue using it.